### PR TITLE
[Master] Made profile (first craft)

### PR DESCRIPTION
**Pull request by alisbi**

### CODE REVIEW

**Review by trevor-jrvs — CHANGES_REQUESTED** [review]
Introduction is too short: I will send you some sample profiles on slack. Please remove any unstarted projects, and make sure there is no remaining Lorem ipsum text.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,43 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Ali Saboohi . Jarvis Consulting
 
+Hi, I'm a recent graduate from Ryerson university with a bachelor's degree in Computer Science. During my four years at Ryerson my main focus was on backend development, machine learning and computer vision where I worked on many projects such as Face Detection application using HOG features, Handwrite Recognition application and created a few websites. Currently, I am a Software and Data Engineer at Jarvis where I am trained to master the tools and frameworks necessary for professional work enviroment such as Git, Docker, Java, Spring boot, RDBMS, Maven etc. In addition to this, I have a passion for the insights data can provide. My experiences taught me how to come up with creative solutions to the real-world problems and I am looking forward to bring my experiences and skills to your team.
```
Comment (jarviscanada) [inline README.md]: I'm a recent graduate from Ryerson University with a bachelor's degree in Computer Science. During my four years at Ryerson, my main focus was on backend development, machine learning and computer vision, where I worked on many projects such as Face Detection application using HOG features, Handwrite Recognition application and created a few websites. Currently, I am a Software and Data Engineer at Jarvis, where I am trained to master the tools and frameworks necessary for a professional work environment such as Git, Docker, Java, Spring boot, RDBMS, Maven etc. In addition to this, I have a passion for the insights data can provide. My experiences taught me how to come up with creative solutions to real-world problems, and I am looking forward to bringing my experiences and skills to your team.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,43 @@
...
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git
+
+**Competent:** Docker, Maven, REST API, JDBC, Spring Boot
```
Comment (jarviscanada) [inline README.md]: Spring Boot, REST API, JDBC, Docker, Maven

File: README.md [hunk 3]
```diff
@@ -1,8 +1,43 @@
...
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_AliSaboohi](https://github.com/jarviscanada/jarvis_data_eng_AliSaboohi)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_AliSaboohi/tree/master/linux_sql)]: The Objective of this project was to design an MVP in order to gather hardware specifications and resource usage of each node in a Linux cluster. In Jarvis, the Linux cluster administration team manages a Linux cluster of ten nodes running Centos7. These nodes are internally connected through a switch and able to communicate through internal IPv4 addresses. The administration team has requested a bash agent program that is installed on each node and gathers the requested data. The collected data is inserted into a Postgres database instance which is pulled from Docker Hub. Moreover, the LCA team will use the data to generate reports for future resource planning purposes. In order to make decision making easier, a queries file has been created for the administration team to see what nodes have the highest memory and number of CPUs, what is their usage over a five minute interval and detecting failures.
```
Comment (jarviscanada) [inline README.md]: Developed a hardware resource monitoring program that collects CPU, RAM, blahahaha in rea-time using Bash scripts, PostgreSQL, Linux, etc. Data is persisted into a PostgreSQL database instance provisioned using a Docker container. Wrote SQL queries to analyze blah blah for blah blah..

File: README.md [hunk 4]
```diff
@@ -1,8 +1,43 @@
...
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_AliSaboohi/tree/master/core_java)]:
+      
+  - Twitter App: 
+  - JDBC App: 
+  - Grep App:
```
Comment (jarviscanada) [inline README.md]: remove

File: README.md [hunk 5]
```diff
@@ -1,8 +1,43 @@
...
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_AliSaboohi/tree/master/core_java)]:
+      
+  - Twitter App: 
+  - JDBC App: 
+  - Grep App: 
+
+
+## Highlighted Projects
+**Clothing Store Web Application**: Designed and Developed Online clothing Web site with Online Shopping facility. using Java and Spring Boot for backend and React Js for Frontend. Frontend developed as SPA application in order to use Client Computer Resources using JavaScript ES6 Version, Vanilla JavaScript, Babel, React-Dom-route, Redux, HTML and CSS, in Backend using Spring Core for dependency Injection and AOP, Spring MVC to develop Rest Endpoints (RestController Bean) and JSON. implementing Data Entity Classes and using  Hibernate, Spring Data JPA, JPA QL for managing Data and store it in MySQL Database
```
Comment (jarviscanada) [inline README.md]: Designed and Developed an online clothing e-commerce website. The frontend was developed using JavaScript ES6 Version, Vanilla JavaScript, Babel, React-Dom-route, Redux, HTML and CSS. The backend was developed using Spring Core for dependency Injection and AOP, Spring MVC to build Rest Endpoints and JSON. Implemented Data Entity Classes using Hibernate, Spring Data JPA, JPA QL to manage and store data in MySQL Database.

File: README.md [hunk 6]
```diff
@@ -1,8 +1,43 @@
...
+
+
+## Professional Experiences
+
+**Software Developer, Jarvis (2020-present)**: Collaborated with a Software and Data engineering team in an Agile enviroment in order to work on various projects using technologies, tools and frameworks such as git, Maven, Java, PostgresSQL, IntelliJ, Docker and bash just to name a few. Attended daily scrum meetings as well as bi-weekly sprint retrospectives. Additionaly, studied and practiced new technologies and better ways to come up with better solutions to the real-world problems.
```
Comment (jarviscanada) [inline README.md]: Collaborated with a Software and Data engineering team in an Agile environment to work on various projects using technologies, tools and frameworks such as git, Maven, Java, PostgresSQL, IntelliJ, Docker, etc. Organized daily scrum meetings as well as bi-weekly sprint retrospectives. Studied and practiced new technologies and better ways to develop better solutions to real-world problems.

File: README.md [hunk 7]
```diff
@@ -1,8 +1,43 @@
...
+
+
+## Education
+**Ryerson university (2018-2021)**, Bachelor of Computer Science, Science and Engineering
```
Comment (jarviscanada) [inline README.md:31]: Edward Wang  Nov 2nd, 2021 at 1:35 PM
https://github.com/jarviscanada/jarvis_data_eng_NarimanAlimuradov
https://github.com/jarviscanada/jarvis_data_eng_JasmeetBrar
https://github.com/jarviscanada/jarvis_data_eng_AlexKirilenko
https://github.com/jarviscanada/jarvis_data_eng_FatimaAnwer
white_check_mark
eyes
raised_hands





Direct Message

Edward Wang  Nov 2nd, 2021 at 1:34 PM
https://github.com/jarviscanada/jarvis_data_eng_BrandonDicicco
https://github.com/jarviscanada/jarvis_data_eng_JohnCao
https://github.com/jarviscanada/jarvis_data_eng_TomasRotbauer
https://github.com/jarviscanada/jarvis_data_eng_TinaLee

**Review by jarviscanada — APPROVED** [review]

**Review by jarviscanada — APPROVED** [review]

**Pull request merged**